templates/common/cloud-platform-alt-dns/files/coredns.yaml

```diff
@@ -40,6 +40,8 @@ contents:
         - "/config"
         - "--out-dir"
         - "/etc/coredns"
+        - "--resolvconf-path"
+        - "/var/run/NetworkManager/resolv.conf"
```

Comment:
Why? Isn't passing /etc/resolv.conf enough?

Reply:
I think /etc/resolv.conf is sufficient. I was going to ask you why /var/run/NetworkManager/resolv.conf was chosen for metal platform. Thanks for clarifying.